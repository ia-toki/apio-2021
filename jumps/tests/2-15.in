209636825ecd2c38299498cd26890ef71dd128c2
200 200
197 194 193 186 173 171 169 168 167 165 162 161 160 157 152 146 145 144 143 142 141 139 137 131 126 124 122 117 112 110 108 107 105 102 101 95 91 90 89 88 85 83 81 76 74 73 72 67 65 58 52 48 47 46 40 38 34 33 29 27 25 24 23 22 21 17 16 15 8 5 4 3 1 2 6 7 9 10 11 12 13 14 18 19 20 26 28 30 31 32 35 36 37 39 41 42 43 44 45 49 50 51 53 54 55 56 57 59 60 61 62 63 64 66 68 69 70 71 75 77 78 79 80 82 84 86 87 92 93 94 96 97 98 99 100 103 104 106 109 111 113 114 115 116 118 119 120 121 123 125 127 128 129 130 132 133 134 135 136 138 140 147 148 149 150 151 153 154 155 156 158 159 163 164 166 170 172 174 175 176 177 178 179 180 181 182 183 184 185 187 188 189 190 191 192 195 196 198 199 200
0 71 72 199
14 18 147 162
62 64 162 172
37 54 87 92
54 61 94 166
9 16 144 153
5 21 190 197
0 10 102 164
0 8 123 139
11 11 160 195
17 56 109 116
38 46 168 181
4 69 141 166
7 9 135 174
2 7 115 182
8 13 99 196
7 22 89 187
0 4 135 195
0 4 149 160
4 37 91 95
43 48 161 184
1 1 102 199
28 35 150 193
4 48 82 147
69 71 172 176
2 3 103 188
9 50 186 199
20 59 142 193
35 38 187 188
7 16 78 79
0 4 198 199
22 50 82 136
0 11 148 169
1 2 96 143
16 36 125 146
38 64 98 198
50 56 122 124
19 38 96 101
23 31 163 190
7 19 131 146
9 18 177 197
18 23 146 198
12 17 124 157
28 69 119 194
28 36 126 177
5 27 187 197
5 5 77 92
5 11 133 196
30 60 163 179
3 4 158 179
18 33 89 164
41 64 110 162
32 44 103 152
43 63 127 153
19 23 100 150
21 66 120 140
2 33 139 176
18 40 162 163
28 52 179 186
8 11 102 136
17 71 85 95
1 3 76 97
45 67 137 196
0 48 124 186
37 41 136 165
3 67 192 193
38 42 170 182
19 61 127 162
8 69 104 160
10 13 194 196
11 11 90 134
7 45 177 186
37 61 185 199
4 5 150 158
3 3 94 148
7 22 89 188
20 22 192 198
54 55 129 145
59 62 121 178
1 2 184 186
32 36 192 192
40 55 198 198
7 8 111 186
15 22 148 169
10 42 126 167
0 0 169 184
1 28 133 186
10 21 176 182
10 21 188 189
1 7 111 125
0 7 125 139
31 53 80 196
2 9 192 199
7 24 91 173
3 13 157 190
38 56 184 184
47 50 78 101
14 15 177 178
7 18 169 169
52 67 173 180
10 12 168 179
47 47 181 193
0 1 99 104
5 10 114 131
0 18 148 164
29 37 172 187
31 56 145 177
29 49 76 116
34 34 85 97
0 5 89 179
0 1 119 126
19 35 125 154
37 37 175 183
40 51 128 168
46 49 87 117
52 61 75 166
8 50 103 124
1 3 102 149
19 19 119 144
26 35 121 146
22 44 192 192
47 48 78 120
5 5 154 188
1 41 76 124
17 22 126 199
5 39 124 131
9 17 132 133
57 66 135 169
13 13 101 125
24 28 180 195
48 60 122 177
27 36 110 163
2 5 186 189
0 3 187 196
8 42 191 197
23 33 197 199
6 10 192 193
4 23 84 186
1 6 151 174
36 46 81 81
23 63 187 189
18 29 195 197
23 53 198 199
46 46 85 141
34 41 112 128
8 9 77 134
61 67 145 165
8 39 149 199
4 12 162 168
7 31 181 187
0 28 187 192
4 51 85 179
21 52 149 187
16 19 191 192
0 4 136 140
26 55 90 172
34 53 142 166
6 16 175 176
5 38 189 194
36 55 103 130
4 10 79 139
14 19 74 155
9 32 157 198
11 22 192 199
14 32 158 163
13 40 83 178
36 60 108 169
1 2 170 195
28 43 191 196
0 27 142 170
32 55 110 114
8 13 104 167
9 31 116 128
23 55 192 197
23 68 104 120
1 12 190 197
9 23 131 195
12 18 171 199
3 32 157 174
0 2 112 197
0 32 175 189
12 70 149 185
45 56 81 143
27 29 195 196
6 22 188 189
0 0 194 199
39 70 73 169
10 31 122 151
25 45 111 128
12 27 116 173
17 38 72 83
17 21 89 191
1 25 139 140
10 13 113 141
41 46 146 179
17 43 165 176
12 53 84 174
54 56 124 153
0 0 107 146
2 68 161 192
