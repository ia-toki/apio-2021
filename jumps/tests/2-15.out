fc5005cd102fffc504bc1615010ebf88823d65f3
OK
1
1
39
-1
1
-1
4
-1
-1
1
1
21
1
1
1
1
1
1
-1
-1
22
1
5
1
46
-1
6
1
35
-1
2
1
-1
-1
1
1
8
-1
6
-1
4
1
-1
1
1
3
-1
1
13
1
1
1
1
5
-1
1
1
2
20
-1
1
-1
12
1
5
4
21
1
1
8
-1
3
29
-1
-1
1
13
16
16
-1
27
34
1
1
1
-1
1
4
7
-1
-1
1
1
1
1
34
-1
8
4
35
1
36
-1
-1
1
14
6
1
-1
1
-1
1
25
3
1
1
1
-1
-1
1
17
1
1
1
1
1
-1
20
-1
16
6
1
1
1
6
18
7
1
-1
-1
15
13
19
1
1
-1
32
1
-1
7
1
1
1
14
-1
1
8
1
3
1
-1
-1
1
9
1
1
1
1
21
1
1
-1
-1
16
1
1
1
1
1
1
1
1
1
20
4
1
1
1
1
1
-1
1
-1
-1
14
3
1
12
-1
1
